packages/config/config/devices/0x020e/zrksw.json

```diff
@@ -0,0 +1,71 @@
+// Domitech Products, LLC ZRKSW
+// Smart LED Light Bulb
+{
+	"manufacturer": "Domitech Products, LLC",
+	"manufacturerId": "0x020e",
+	"label": "ZRKSW",
+	"description": "Smart LED Light Bulb",
+	"devices": [
+		{
+			"productType": "0x4c42",
+			"productId": "0x3133",
+			"zwaveAllianceId": 1443
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Dim Level When Powered On",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Full brightness",
+					"value": 0
+				},
+				{
+					"label": "Last dim level",
+					"value": 1
+				}
+			]
+		},
+		"9": {
+			"label": "Dim Step Level",
+			"description": "Defines the number of steps necessary to reach full brightness.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Dim Step Timing",
+			"description": "Timing of steps",
```

Comment:
any unit here?

Reply:
There is none specified in the manual or elsewhere.